uaaextras/templates/email/body.html

```diff
@@ -37,7 +37,15 @@
         </p>
         <p>
             Thank you,<br />
-            The {{branding.company_name}} team
+            {{branding.company_name}} Team
```

Comment:
I can't access the cloud.gov style guide at the moment, but if you haven't already done so, it'd be good to doublecheck whether we capitalize or lowercase `Team`.

Reply:
We don't have specific guidance on that (https://github.com/18F/cg-product/blob/master/ContentGuide.md). When we don't have specific guidance on something, we default to the 18F Content Guide (https://content-guide.18f.gov/), which recommends being consistent and judicious about capitalizing noun phrases. We default to lowercase unless we need uppercase. So, "cloud.gov team" is more suitable.

Reply:
@rememberlenny you should lowercase team